### PR TITLE
Initial project setup

### PR DESCRIPTION
**Pull request by AnaMendozaP**
Initial project setup

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @holgiosalos @valeryibarra @aperdomob @manuelq12 @JuanDelahozMiranda
\ No newline at end of file
+@holgiosalos @valeryibarra @aperdomob @manuelq12 @JuanDelahozMiranda
```
Comment (juanjo44) [inline .github/CODEOWNERS]: Here you need to add the: * symbol at the begining and also add an End Of File(line break)

File: package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "type": "git",
+    "url": "git+https://github.com/AnaMendozaP/cypress-training.git"
+  },
+  "keywords": [
+    "ui-testing",
+    "dojo",
+    "practice",
+    "cypress"
+  ],
+  "author": "AnaMendoza < ana.mendozap@perficient.com >",
```
Comment (juanjo44) [inline package.json]: if you want you can add your github user after the last: >

File: .github/CODEOWNERS (hunk 1 above)
Comment (ana-om) [reply .github/CODEOWNERS]: You can do this to add the EOF line automatically [Method I](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files).

File: package.json [hunk 3]
```diff
@@ -16,7 +16,7 @@
     "practice",
     "cypress"
   ],
-  "author": "AnaMendoza < ana.mendozap@perficient.com >",
+  "author": "AnaMendoza < ana.mendozap@perficient.com > https://github.com/AnaMendozaP",
```
Comment (juanjo44) [inline package.json:19]: just AnaMendozaP but it's ok

**Review by juanjo44 — APPROVED** [review]

**Review by ana-om — APPROVED** [review]

**Review by manuelq12 — APPROVED** [review]

**Pull request merged**